hexkit/protocols/eventsub.py

```diff
@@ -55,4 +54,18 @@ def consume(self, *, payload: JsonObject, type_: Ascii, topic: Ascii) -> None:
             topic (str): Name of the topic to publish the event to.
         """
         check_ascii(type_, topic)
+        await self._consume_validated(payload=payload, type_=type_, topic=topic)
+
+    @abstractmethod
+    async def _consume_validated(
+        self, *, payload: JsonObject, type_: Ascii, topic: Ascii
+    ) -> None:
+        """
+        Receive a pre-validated event of interest and process it according to its type.
```

Comment:
This description is a bit misleading, you might think that the event itself is or was validated.

"Receive and process an event with already validated topic and type."?

Reply:
Agreed.

Reply:
✅